### ISSUE TITLE
Provide browser compatible javascript package via CDN

### ISSUE THREAD

**Issue by armaniferrante** [solana-foundation/anchor#3]
Would be nice to  use the @project-serum/anchor package directly in the browser without a build step, i.e., via `<script>`.

**Comment by jackrobsonalpha** [comment]
is this possible with a build step at the moment?

**Comment by armaniferrante** [comment]
Possible once we merge https://github.com/project-serum/anchor/pull/1108.

**Comment by jeduan** [comment]
Thinking out loud would the idea be that we create a full bundle with all the required dependencies? or would we expect users to, say add @solana/web3 in a separate tag?

**Comment by armaniferrante** [comment]
A separate tag would be nice if there's any reason to have different versions (or even web3 libraries in the future), though I can't think of any examples where that would be useful atm.

**Comment by MrMizz** [comment]
any update here? 🙇

**Comment by rmdorsey** [comment]
Was a browser bundle created for anchor?  I can't find it...

**Comment by MrMizz** [comment]
> Was a browser bundle created for anchor? I can't find it...

nope. you'll have to bundle yourself. [here's my implementation for reference](https://github.com/bigtimetapin/somos-solana/blob/develop/assets/js/rollup.config.js)

**Comment by rmdorsey** [comment]
> 

It looks like somebody already did it?  https://unpkg.com/@project-serum/anchor@latest

**Comment by rmdorsey** [comment]
this didn't work for me: https://unpkg.com/@project-serum/anchor@latest

@MrMizz , I'll try it your way.  What's the process for creating a browser bundle?  Is there a tutorial you can recommend?

**Comment by MrMizz** [comment]
@rmdorsey like yourself I did not have experience with browser bundles until I started developing with anchor-lang.
I found [rollup.js](https://rollupjs.org/guide/en/) by reading the anchor codebase. that's what they use for bundling their assets as well. read the docs I linked. I went from nothing to a successful build in a couple hours. you can use my previous link to my src as a reference implementation.

**Comment by SkyYap** [comment]
> > 
> 
> It looks like somebody already did it? https://unpkg.com/@project-serum/anchor@latest

Hi, I tried to use anchor on WebGL and found out WebGL can't install npm. For @solana/web3.js, I have successfully import the browser bundle following their [tutorial](https://docs.solana.com/developing/clients/javascript-api) in Getting Started -> Installation -> Bundle. 

I spot Browserify unpky skypack.dev tools provide feature that turn npm packages to browser bundle but all of these file and can't load on my browser. Tried the above unpkg too and it doesn't work too. Anyone can help me?

**Comment by Swader** [comment]
I'm going to lend my voice here for the side of decentralization and against complexity. A bundle, or at least an easy way to bundle anchor's client, would be nice to have. Right now ESBuild has problems with Buffer, for example. A package on-release would alleviate many, many headaches.

**Comment by jacobcreech** [comment]
Closing for now. I will reopen if this comes up as priority
